Library/Formula/cctools.rb

```diff
@@ -12,6 +12,7 @@ class Cctools < Formula
   end
 
   bottle do
+    does_not_need_relocation
     cellar :any
```

Comment:
I may just remove `does_not_need_relocation` altogether, as that was mainly there for the previous system (hooking in `cctools`). Do you still want me to make this change?